Pegasus/sources/Math.cpp

```diff
@@ -384,8 +392,11 @@ void HalfEdgeDataStructure::IntializeHalfEdge(
 }
 
 double pegasus::math::LineSegmentPointDistance(
-    glm::dvec3 const& lineStart, glm::dvec3 const& lineEnd, glm::dvec3 const& point
+    glm::dvec3 const& lineStart, glm::dvec3 const& lineEnd, glm::dvec3 point
```

Comment:
I am changing the value inside the function, makes perfect sense to copy it in place then.